pino.js

```diff
@@ -400,9 +408,9 @@ function genLog (z) {
       n = [b, c, d, e, f, g, h, i, j, k]
       l = 1
 
-      if (m.method && m.headers && m.socket) {
+      if (isHttpRequest(m)) {
         m = mapHttpRequest(m)
-      } else if (m.statusCode) {
+      } else if (isHttpResponse(m)) {
```

Comment:
Yes. There is a maximum number of functions that can be inlined automatically by V8.